source/docs/software/commandbased/command-based-changes.rst

```diff
@@ -16,7 +16,7 @@ The overall structure of the command-based framework has remained largely the sa
 Commands and Subsystems as Interfaces
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-``Command`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Command.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc2_1_1_command.html>`__) and ``Subsystem`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Subsystem.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc2_1_1_subsystem.html>`__) are both now interfaces as opposed to abstract classes, allowing advanced users more potential flexibility.  ``CommandBase`` and ``SubsystemBase`` abstract base classes are still provided for convenience, but are not required.  For more information, see :doc:`commands` and :doc:`subsystems`.
+``Command`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Command.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc2_1_1_command.html>`__) and ``Subsystem`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Subsystem.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc2_1_1_subsystem.html>`__) are both now interfaces as opposed to abstract classes, allowing advanced users more potential flexibility.  ``CommandBase`` and ``SubsystemBase`` abstract base classes are still provided for convenience, but are not required.  For more information, see :doc:`commands` and :doc:`subsystems`. // TODO this isn't correct anymore
```

Comment:
update per todo

Reply:
That article will be deleted by #2287, i'll just remove my changes to the file.